src/common/components/forms/ButtonGray.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Button, ButtonProps, Text } from '@chakra-ui/react';
+
+type CustomButtonProps = {
+  children: React.ReactNode;
+} & ButtonProps;
+
+export default function CustomButtonGray({
```

Comment:
Maybe merge this with `CustomButton` component

Reply:
This has been added as a variant, and the `GrayButton` component has been deleted